cardano-node/src/Cardano/Node/TraceConstraints.hs

```diff
@@ -1,35 +1,38 @@
 {-# LANGUAGE ConstraintKinds  #-}
 {-# LANGUAGE DataKinds        #-}
+{-# LANGUAGE ImportQualifiedPost #-}
 {-# LANGUAGE FlexibleContexts #-}
 module Cardano.Node.TraceConstraints (TraceConstraints) where
 
-import           Prelude (Show)
+import Prelude (Show)
 
-import           Data.Aeson
+import Data.Aeson
 
-import           Cardano.BM.Tracing (ToObject)
-import           Cardano.Logging (LogFormatting)
-import           Cardano.Node.Queries (ConvertTxId, GetKESInfo (..),
-                     HasKESInfo (..), HasKESMetricsData (..), LedgerQueries)
+import Cardano.BM.Tracing (ToObject)
```

Comment:
Style regression here

Reply:
Undone the change to `ImportQualifiedPost` after discussion with @Jimbo4350